tests/test_CFDataset.py

```diff
@@ -181,6 +187,21 @@ def test_dim_axes2(tiny_dataset, expected):
     assert tiny_dataset.dim_axes() == expected
 
 
+# Tests for all dimensions - may differ between datasets.
+@mark.parametrize('tiny_dataset, expected', [
+    ('gcm', {'T': 'time', 'X': 'lon', 'Y': 'lat'}),
+    ('downscaled', {'T': 'time', 'X': 'lon', 'Y': 'lat'}),
+    ('hydromodel_gcm', {'T': 'time', 'X': 'lon', 'Y': 'lat'}),  # why isn't depth in this list?
+    ('climo_gcm', {'T': 'time', 'X': 'lon', 'Y': 'lat'}),
+], indirect=['tiny_dataset'])
+def test_axes_dim(tiny_dataset, expected):
+    assert tiny_dataset.axes_dim() == expected
+
+
+# TODO: Obtain a test file with reduced dimensions and test reduced_dims with it.
+# Request sent to Stephen Sobie and Trevor Murdock 2017-06-02 for such a file.
```

Comment:
Yeah, they may not know of any. I'm pretty sure that David Bronaugh would not have put compressed dimension handling in if he hadn't seen it already, so maybe some old CMIP3 files used it? I'd be surprised if Trevor or Stephen have seen (or remembered) them.